subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/AbstractGradleExecuter.java

```diff
@@ -134,6 +134,7 @@ public static void propagateSystemProperty(String name) {
 
     protected boolean noExplicitTmpDir;
     protected boolean noExplicitNativeServicesDir;
+    protected boolean noFullDeprecationStackTrace;
```

Comment:
Please call this field `fullDeprecationStackTrace` and initialize it to true. `eagerClassLoaderCreationChecksOn` is a better template for a flag like that. Double negation makes my head hurt.